src/librustdoc/html/static/main.js

```diff
@@ -2039,11 +2039,9 @@ function defocusSearchBar() {
      * @return {[string]}      [The resulting plaintext]
      */
     function convertHTMLToPlaintext(html) {
-        var dom = new DOMParser().parseFromString(
-            html.replace('<code>', '`').replace('</code>', '`'),
-            'text/html',
-        );
-        return dom.body.innerText;
+        var x = document.createElement("div");
+        x.innerHTML = html;
```

Comment:
This drops the `<code>`handling, was that intentional?

Reply:
No it was not, thank you! I actually noticed that I had removed the handling by mistake, but then I forgot to add it back 🤦

Reply:
Well, I personally think it's better this way but it's up to you. :)